doc/dev/osd_internals/manifest.rst

```diff
@@ -314,8 +290,6 @@ Operations:
 
         rados -p base_pool tier-flush <obj-name>
 
-  Included in the user_modify=false whitelist, does not trigger a clone.
```

Comment:
I'm not sure why you are removing these bits, do you disagree with them?  Have you added similar explanations elsewhere?

Reply:
In particular, I think the portions explaining interactions with user_modify/clone, side effects, and return codes are pretty critical.  If they are incorrect, I'd prefer that you correct them rather than remove them.

Reply:
The purpose of this document is to describe overall direction of tiering/dedup (am I wrong). So, I deleted comments because these are about specific items for implementation. But, I agree on the deleted comment and will apply it when I make the PR related to tiering/dedup.

Reply:
No.  I've broken this up into into several files specifically to allow much more detailed information on how the implementation works, what's missing, and what needs to happen next.  I think this document should have enough information for any new contributor to quickly come up to speed on how the operations work and what needs to be done next.  I think these notes on how the operations interact with user_modify, etc are pretty crucial in that regard, and I'd like you to ensure that they are correct.